components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -224,9 +234,46 @@ def get_clp_connection_params_and_type(self, disable_localhost_socket_connection
         return connection_params_and_type
 
     def dump_to_primitive_dict(self):
-        d = self.model_dump(exclude={"username", "password"})
+        d = self.model_dump(
+            exclude={"username", "password", "privileged_username", "privileged_password"}
```

Comment:
note the `root_password` should be excluded too
```suggestion
            exclude={"root_password", "admin_username", "admin_password", "username", "password"}
```